tpl/tplimpl/embedded/templates/opengraph.html

```diff
@@ -2,8 +2,15 @@
 <meta property="og:description" content="{{ with .Description }}{{ . }}{{ else }}{{if .IsPage}}{{ .Summary }}{{ else }}{{ with .Site.Params.description }}{{ . }}{{ end }}{{ end }}{{ end }}" />
 <meta property="og:type" content="{{ if .IsPage }}article{{ else }}website{{ end }}" />
 <meta property="og:url" content="{{ .Permalink }}" />
-{{- with .Params.images }}{{ range first 6 . }}
+{{ with $.Params.images }}
```

Comment:
The fix is correct and good, but I suspect this would be easier to understand (and shorter) if you used the `Param` method, e.g:

```
{{ with $.Param "images" }}
```

The above will first look in page, then in site.

Reply:
That is great, thank you! I changed it to use this way, much better.